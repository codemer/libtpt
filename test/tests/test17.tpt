@set(arr, "orange", "apple", "pear", "banana")\
@set(i, 1)\
@size(arr)
0: ${arr[0]}
2: ${arr[2]}
1: ${arr[1]}
3: ${arr[3]}
1000: ${arr[1000]}
\@concat(i+1, "hi") gives @concat(i+1, "hi")
\@eval(i) gives @eval(i)
i=${i}: ${arr[i]}
${arr[i+2]}
