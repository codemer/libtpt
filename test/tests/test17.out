4
0: orange
2: pear
1: apple
3: banana
1000: 
@concat(i+1, "hi") gives 2hi
@eval(i) gives 1
i=1: apple
banana
